edb/lib/cfg.edgeql

```diff
@@ -192,6 +194,13 @@ ALTER TYPE cfg::AbstractConfig {
             'Recompile all cached queries on DDL if enabled.';
     };
 
+    CREATE PROPERTY _query_cache_mode -> cfg::QueryCacheMode {
```

Comment:
Are we sure we want the leading `_`?

Reply:
I guess so, like we'll probably drop this once cache is stable? Also I think it needs more work to be a public config, e.g. it's not yet clearing cache if changed on system level, or if we really want to allow all modes of cache exist at once by allowing per-statement cache policy (it works implicitly, just confusing).

Reply:
Hm, ok.
I sort of expect we'll keep it, though?

Reply:
I'll drop the leading underscore - we just don't have to advertise it now.